Detectors/EMCAL/calibration/include/EMCALCalibration/EMCALCalibExtractor.h

```diff
@@ -40,12 +40,14 @@ class EMCALCalibExtractor
   using boostHisto = boost::histogram::histogram<std::tuple<boost::histogram::axis::regular<double, boost::use_default, boost::use_default, boost::use_default>, boost::histogram::axis::integer<>>, boost::histogram::unlimited_storage<std::allocator<char>>>;
 
  public:
-  EMCALCalibExtractor() = default;
+  EMCALCalibExtractor()
+  {
+    LOG(info) << "initialized EMCALCalibExtractor";
+    mGeometry = o2::emcal::Geometry::GetInstanceFromRunNumber(300000);
+    NCELLS = mGeometry->GetNCells();
```

Comment:
All-capital is usually used for const, for members please use mNcells (members starting with "m")